R/processes.R

```diff
@@ -322,7 +322,7 @@ aggregate_time = Process$new(
     # prepare paths
     udf_transaction = prepare_udf_transaction(user,script,job$job_id)
     
-    udf_transaction$prepareExportData(collection,export_type=c("json","file"))
+    udf_transaction$prepareExportData(collection,export_type=c("json","file"), dim_mod = names(which(self$modifier == TRUE)))
```

Comment:
during debugging i found that in the context of execution of the graph `sel$modifier` is not available. Also the dimension modifier consists of two dimension lists (for adding and removing dimensions)